ChangeLog.md

```diff
@@ -3,6 +3,309 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
```

Comment:
"and introduced system-control circle colours, only one empire can control/supply a system"

It's a very important and significant change, it needs explaining.